nikola/post.py

```diff
@@ -745,11 +745,16 @@ def section_color(self, lang=None):
 
     def section_link(self, lang=None):
```

Comment:
Why isn't this simply using the `index_section_path` path handler of the `indexes` plugin?